riscv-asm.md

```diff
@@ -183,6 +183,26 @@ Directive    | Arguments                      | Description
 .balign      | b,[pad_val=0]                  | byte align
 .zero        | integer                        | zero bytes
 
+### <a name=.option></a> `.option`
+
+#### `rvc`/`norvc`
+
+Enable/disable C-extension for the following code region.
+
+#### `pic`/`nopic`
+
+Set the code model to PIC (position independent code) or non-PIC, this will
+affect how `la` expands.
```

Comment:
Nit: "Set the code model to PIC (position independent code) or non-PIC. This will affect the expansion of the `la` pseudoinstruction."
Is there some place we can link to that details how `la` is expanded with the various models?

Reply:
Add reference to pseudoinstructions